src/renderer/src/components/home/Series/SeriesCreateModal/SeriesCreateModal.tsx

```
@@ -0,0 +1,82 @@
+import { color } from '@renderer/design/styles'
+import { useOutsideClick } from '@renderer/hooks'
+import { flex } from '@renderer/utils'
+import { useState } from 'react'
+import { styled } from 'styled-components'
+
+interface Props {
+  isOpen: boolean
+  onClose: () => void
+}
+
+const SeriesCreateModal = ({ isOpen, onClose }: Props) => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)